hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/fsck/TestMissingContainerTask.java

```diff
@@ -93,12 +82,13 @@ public void testRun() throws Exception {
     when(containerManagerMock.getContainerReplicas(new ContainerID(3L)))
         .thenReturn(Collections.emptySet());
 
-    List<MissingContainers> all = missingContainersTableHandle.findAll();
+    MissingContainersDao missingContainersDao =
+        getDao(MissingContainersDao.class);
+    List<MissingContainers> all = missingContainersDao.findAll();
```

Comment:
Do we need to get the dao again?  `missingContainersTableHandle` already has an instance.

Reply:
Thank you @adoroszlai. Fixed it.